marketplace-ui/src/app/modules/product/product-detail/product-detail-information-tab/product-detail-information-tab.component.spec.ts

```diff
@@ -1,30 +1,105 @@
-import { ComponentFixture, TestBed } from '@angular/core/testing';
 
+import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { ProductDetailInformationTabComponent } from './product-detail-information-tab.component';
-import { MOCK_PRODUCT_DETAIL } from '../../../../shared/mocks/mock-data';
+import { of } from 'rxjs';
+import { SimpleChanges } from '@angular/core';
+import { ProductDetailService } from '../product-detail.service';
+import { LanguageService } from '../../../../core/services/language/language.service';
+import { ProductDetail } from '../../../../shared/models/product-detail.model';
 import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { ExternalDocument } from '../../../../shared/models/external-document.model';
 
-describe('InformationDetailComponent', () => {
+const TEST_ID = 'portal';
```

Comment:
can we reuse MOCK_EXTERNAL_DOCUMENT since I saw some mock values here are same as it